modules/openapi-generator/src/main/resources/cpp-pistache-server/model-header.mustache

```diff
@@ -35,12 +35,10 @@ public:
     /// <summary>
     /// {{description}}
     /// </summary>
-    {{#isContainer}}{{{dataType}}}& {{getter}}();
-    {{/isContainer}}{{^isContainer}}{{{dataType}}} {{getter}}() const;
-    void {{setter}}({{{dataType}}} const{{^isPrimitiveType}}&{{/isPrimitiveType}} value);
-    {{/isContainer}}{{^required}}bool {{nameInCamelCase}}IsSet() const;
-    void unset{{name}}();
-    {{/required}}
+    {{{dataType}}}{{#isContainer}}&{{/isContainer}} {{getter}}(){{^isContainer}} const{{/isContainer}};
```

Comment:
It is still funny if this setter is available only for non required fields.

I think we need to add it.

Reply:
With my changes there are setters for both (please check the second commit, where I update the samples, `PhotoUrls` is required and has a setter

Reply:
My mistake, only the unset/IsSet is available for non required fields.

Reply:
I was thinking you would do same approach for all.
But the setter is there for all for sure.